services/ux-backend/handlers/utils.go

```diff
@@ -0,0 +1,20 @@
+package handlers
+
+import (
+	"fmt"
+	"os"
+)
+
+var namespace string
+
+// returns namespace found in env value, will panic if value is empty
+func GetOperatorNamespace() string {
```

Comment:
This is not an operator so why do we call the function `GetOperatorNamespace`? 
```suggestion
func GetPodNamespace() string {
```